hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/validators/TokenSupplyChangeOpsValidator.java

```diff
@@ -159,9 +159,10 @@ private void validateCommon(
      */
     private void validateCounts(
             final int nftCount, final long fungibleCount, final boolean nftsAreEnabled, final long maxBatchSize) {
-        if (fungibleCount > 0) {
-            validateTrue(fungibleCount <= maxBatchSize, BATCH_SIZE_LIMIT_EXCEEDED);
-        }
+        // commented out below, checked with Michael T, nfts.maxBatchSizeMint is not for fungibleCount.
```

Comment:
If this is not needed, we should remove the code entirely.

Reply:
I left this for @Neeharika-Sompalli to review when she is back.